src/blueapi/service/runner.py

```diff
@@ -141,9 +141,9 @@ def _rpc(
     **kwargs: Any,
 ) -> T:
     mod = import_module(module_name)
-    func: Callable[P, T] = _validate_function(
+    func: Callable[..., T] = _validate_function(
         mod.__dict__.get(function_name, None), function_name
-    )
+    )  # type: ignore
```

Comment:
Should: Does marking `args` and `kwargs` as `P` solve this problem?

See paramspec docs: https://docs.python.org/3/library/typing.html

Reply:
solved, thanks for the tip!